src/kOS/Module/kOSProcessor.cs

```diff
@@ -405,19 +401,17 @@ private IEnumerable<VolumePath> BootDirectoryFiles()
         public void InitObjects()
         {
             shared = new SharedObjects();
-            CreateFactory();
 
             shared.Vessel = vessel;
             shared.Processor = this;
             shared.KSPPart = part;
             shared.UpdateHandler = new UpdateHandler();
             shared.BindingMgr = new BindingManager(shared);
-            shared.Interpreter = shared.Factory.CreateInterpreter(shared);
+            shared.Interpreter = new Screen.ConnectivityInterpreter(shared);
```

Comment:
Correct.  The only reason I didn't just merge the `ConnectivityInterpreter` with the `Interpreter` was to keep all of this logic separated.  Really, there isn't a major reason to keep the inheritance tree anymore.